docs/pages/[...slug].vue

```diff
@@ -20,9 +20,9 @@ const { data: surround } = await useAsyncData(`${route.path}-surround`, () => qu
 )
 
 useSeoMeta({
-  titleTemplate: '%s - Tailwind CSS module for Nuxt',
+  titleTemplate: '%s - Nuxt Tailwind',
```

Comment:
I think "Nuxt Tailwind" is forbidden by Tailwind CSS trademark - https://tailwindcss.com/brand#trademark-usage-agreement

Reply:
This is how we've branded all our modules so I'll let @Atinux change it back if needed.

Reply:
While @Atinux also gets to this comment, I'll link the discussion from the insiders meet too - https://github.com/nuxt/insiders/discussions/69#discussioncomment-6637547 where I mention about the third-party libraries' guidelines. The module naming would need to be compliant, along with the logo being compliant (either not having a logo or using Nuxt and Tailwind CSS logos side-by-side). To keep the logo in the square form factor, I thought of putting the winds through the mountains but not 100% sure if that works or not (execution isn't great since I'm not a graphic designer), but lets really clarify this (about the naming and logo usage) please.